main.js

```diff
@@ -113,5 +113,58 @@ $(document).ready(function(){
     'students', 'colors' and 'values' arrays.
 
     For example, a function like createList(title, array).
+
+
+
+
   */
-})
\ No newline at end of file
+
+ 
+ var main =$('main')
+
+ var sections = $('<section/>')
+ var ul = $('<ul/>')
+ var div = $('<div/>')
+ main.append(sections)
+ sections.append(div)
+ div.append(ul)
+
+ 
+ function renderHouse (){
+ 
+  for (i=0 ; i<hogwarts.houses.length ; i++){
+    var house = hogwarts.houses[i];
+    var $img = $('<img/>').attr('src' , house.img)
+    var $name = $('<h2/>').text(house.name)
+    var $founder = $('<p/>').text('founder: ' + house.founder)
+    var $mascot = $('<p/>').text('mascot: ' + house.mascot)
+    var $values = $('<h3/>').text('Values ')
+    var $valuesText = $('<p/>').text(house.values.join("\r\n"))
+    var $colors = $('<h3/>').text('Colors ')
+    var $colorsText =  $('<p/>').text(house.colors.join("\r\n"))
+    var $students = $('<h3/>').text('Students ')
+    var $studentsText = $('<p/>').text(house.students.join("\r\n"))
+    
+   
+  ul.append($img)
```

Comment:
I love that you have the variables in one place and the appends down in another place.  One cool thing you could do to make this even cleaner is use a single `.append` to append them all at once.

Like this:
```js
ul.append($img, $name, $founder, $mascot, (etc))
```

then you can make this a crazy one-liner that gets a lot done.